ScoutSuite/output/data/html/partials/aws/services.dynamodb.regions.id.tables.html

```diff
@@ -0,0 +1,29 @@
+
+<!-- Table partial -->
+<script id="services.dynamodb.regions.id.tables.partial" type="text/x-handlebars-template">
+  {{#unless scout2_link}}
```

Comment:
What's that for? I've never seen that!

Reply:
To be quite frank, I only put it there because I wanted to keep the same format as previous files and that section is present in many AWS partials, e.g. : services.cloudformation.regions.id.stacks.html, services.cloudtrail.regions.id.trails.html, services.cloudwatch.regions.id.alarms.html, services.ec2.regions.id.volumes.html, services.ec2.regions.id.vpcs.id.images.html and so on...